torch_xla/experimental/plugins.py

```diff
@@ -84,21 +85,12 @@ def default() -> DevicePlugin:
 
 def register_plugin(name: str, device_plugin: DevicePlugin):
   _plugin_registry[name.upper()] = device_plugin
-  torch_xla._XLAC._register_pjrt_plugin(
-      name, device_plugin.library_path(), device_plugin.client_create_options(),
-      device_plugin.requires_xla_coordinator())
+  torch_xla._XLAC._register_pjrt_plugin(name, device_plugin)
 
 
 def register_installed_plugins():
```

Comment:
pytorch/xla/plugins/cuda/README.md gives an example of how to use `register_plugin`. I wonder under what circumstance do we use `register_installed_plugins()` (I see it's used when we `import torch_xla`. Does it mean if we set XLA_REGISTER_INSTALLED_PLUGINS, then we don't have to use `register_plugin` anymore?)

Reply:
I don't expect users to ever call either of these themselves. As long as plugin authors set the entrypoint correctly in their package, registration will happen in the background. Otherwise, plugin authors may add `register_plugin` to their module and run it on import (similar to how `torch.distributed` backend registration works).